### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by merspieler** [qutebrowser/qutebrowser#7546] [General] [answered]
Thank you for making this wonderful program which I really enjoy using.

**Comment by The-Compiler** [comment, marked as answer]
You're welcome! :sparkles:  Converting this to a discussion though, since issues are intended for something that's actionable in some way or another :slightly_smiling_face:

**Reply by iambumblehead** [reply]
thank you from me also Qutebrowser is amazing and wonderful to use and my favorite browser
